x-pack/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -21,12 +21,12 @@ import { PageTitle, pageTitleContent } from './components/page_title';
 import { HeaderActions } from './components/header_actions';
 import { AlertSummary, AlertSummaryField } from './components/alert_summary';
 import { CenterJustifiedSpinner } from '../../components/center_justified_spinner';
+import { FeedbackButton } from './components/feedback_button';
```

Comment:
This re-arrangement of imports doesn't seem to have any benefit 🤔

Reply:
I like to have all the imports of components together.

Reply:
Ah, OK